config.go

```diff
@@ -48,7 +52,7 @@ type Config struct {
 	Verbose bool
 
 	// The default context set on each message sent by the client.
-	DefaultContext Context
+	DefaultContext *Context
```

Comment:
This was changed to a pointer for consistency with other types that now embed a `*Context` as well. Using a pointer instead of the value is necessary to have the `omitempty` tag work (zero-value structs are not considered *empty*).